src/pages/dev/send-tokens/interchain-tokens.mdx

```diff
@@ -30,36 +52,18 @@ function interchainTransfer(
 ) external payable;
 ```
 
-### Custom Tokens
-If you want more features than the [standardized](#standardized) token provides, you can create a [custom](#custom) token. You may want a customized token if you want to customize minting policies, ensure ownership or control of the token by a DAO, create rate limits, or build any other custom logic into your token.
+### Create a new custom Interchain Tokens
 
-You will need to [build this token yourself](https://docs.alchemy.com/docs/how-to-create-an-erc-20-token-4-steps). Make sure your new token implements the `IInterchainToken` interface so you can offer `interchainTransfer` and `interchainTransferFrom` methods directly on your token. 
+If you want more features than the [standardized](../reference/glossary/#standardized) token provides, you can create a [custom](../reference/glossary/#custom) token. You may want a customized token to customize minting policies, ensure ownership or control of the token by a DAO, create rate limits, or build any other custom logic into your token.
 
-You can try our [sample custom token](https://remix.ethereum.org/axelarnetwork/axelar-docs/blob/main/public/samples/interchain-token-iinterchaintoken.sol) as a starting point.  This token self-registers with the Interchain Token Service, and can be deployed to multiple chains. 
+You will need to [build this token yourself](https://docs.alchemy.com/docs/how-to-create-an-erc-20-token-4-steps). Make sure your new token implements the [`IInterchainToken`](https://github.com/axelarnetwork/interchain-token-service/blob/main/contracts/interfaces/IInterchainToken.sol) interface so you can offer [`interchainTransfer`](https://github.com/axelarnetwork/interchain-token-service/blob/main/contracts/interfaces/IInterchainToken.sol#L20) and [`interchainTransferFrom`](https://github.com/axelarnetwork/interchain-token-service/blob/main/contracts/interfaces/IInterchainToken.sol#L37) methods directly on your token. 
```

Comment:
Implementing these methods isn't required, and I think we should actually stop pushing it, even though they make tokens LOOK like standardized tokens, they won't be. This would mean that we recommend folks use the Token Managers to transfer, rather than the token itself. This Ability to use these methods yourself maybe should become a footnote at the bottom of the doc.